### PR TITLE
Feature/pic

### PR DESCRIPTION
**Pull request by scolalongo**
Add pic feature.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 ### IDE files ###
 /.idea
 *.iml
+
+.env
```
Comment (MarcAntoine-Arnaud) [inline .gitignore]: missing return line

File: README.md [hunk 2]
```diff
@@ -12,3 +12,82 @@
...
+
+The command `make docker-build` will build an image named `mediacloudai/rdf_worker`.
+
+The command `make push-docker-registry` will logged in and push the built image in the official docker registry. The login must be set with the following environment variables:
+
+| Variable name           | Default value              | Description                                                  |
+|-------------------------|----------------------------|--------------------------------------------------------------|
+| `DOCKER_REGISTRY_LOGIN` |                            | User name used to connect to the docker registry.            |
+| `DOCKER_REGISTRY_PWD`   |                            | Password used to connect to the docker registry.             |
+| `RUSTTOOLS_DOCKER_IMG`  |                            | Name of RUST image containing CLIPPY, TARPAULIN and RUSTFMT. |
```
Comment (MarcAntoine-Arnaud) [inline README.md]: can we use RUST_TOOLS_DOCKER_IMG var name ?

File: .gitignore (hunk 1 above)
Comment (scolalongo) [reply .gitignore]: Done.

File: README.md (hunk 2 above)
Comment (scolalongo) [reply README.md]: Yes, this variable is used in Gitlab-CI. So I moved the description is the corresponding section of the README.

**Pull request merged**